src/pages/PrivacyPolicyPage/index.tsx

```diff
@@ -0,0 +1,225 @@
+import { useEffect, type FC } from 'react';
+import { useNavigate } from 'react-router';
+import { Email } from '@shared/ui/Email';
+
+export const PrivacyPolicyPage: FC = () => {
+  const navigate = useNavigate();
+
+  const onGoBackClick = (): void => {
+    if (window.history.length > 1) {
+      navigate(-1);
+    } else {
+      navigate('/');
```

Comment:
```suggestion
      navigate(window.history.length > 1 ? -1 : '/');
```

Reply:
can't use the ternary operator, it causes an error